src/_includes/common/configuration-basics.md

```diff
@@ -0,0 +1,184 @@
+{% comment %}
+Guideline: This page is comprehensive; it is stored in the common folder, nested under _includes/common. To use, 
+
+1. Add a folder with the name of the platform you are documenting to the _documentation/sdks structure (for example, _documentation/sdks/javascript) 
+2. Create a copy of "basics.md" file in _documentation/sdks/<platform-name> 
+3. Create the defined `include` statements and add them to the configuration-basics.md file
+4. Note that each of these terms is wrapped in an "if/then" statement; this is because not all of these options are included for every SDK. For those terms you do not wish to include, add the command `hide_<option>=true` in the SDK-specific page.
+
+If you have questions, please ask Fiona or Daniel. 
+
+**The objective for this page is that a developer can view easily the configuration options for the SDK; each page _must_ have a description that includes a summary of what the page provides to the developer. Simply linking the page is insufficient.**
+{% endcomment %}
+
+The {{ include.sdk_name }} SDK is configurable using a variety of options. These options are set when the SDK is first initialized, passed to the `init()` as an object:
+
+{{ include.config_basic_content }}
+
+These options can be read from an environment variable or your `~/.sentryclirc` file (`SENTRY_DSN`, `SENTRY_ENVIRONMENT`, `SENTRY_RELEASE`) are read automatically.
```

Comment:
`.sentryclirc` is only meant for `sentry-cli` and none of the SDKs use it.

Reply:
Oh, good catch. Thank you.